### ISSUE TITLE
Endless log entry generation in CLI

### ISSUE THREAD

**Issue by tomasfejfar** [pehapkari/pehapkari.cz#77]
When starting sculpin with:

```
composer update
vendor/bin/sculpin generate --watch --server
```

I get endless stream of messages:

```
λ vendor\bin\sculpin generate --watch --server
Didnt find at least one of this type : posts
Detected new or updated files
Generating: 100% (38 sources / 0.01 seconds)
Converting: 100% (38 sources / 0.07 seconds)
Formatting: 100% (38 sources / 0.01 seconds)
Processing completed in 0.20 seconds
Starting Sculpin server.
Server is running at http://localhost:8000
Quit the server with CONTROL-C.
Didnt find at least one of this type : posts
Didnt find at least one of this type : posts
Didnt find at least one of this type : posts
...
Didnt find at least one of this type : posts
Didnt find at least one of this type : posts
Didnt find at least one of this type : posts
```

**Comment by TomasVotruba** [comment]
Thanks for reporting. This is due to [PHP7_Sculpin](https://github.com/Symplify/PHP7_Sculpin) being in development. 

Will be fixed in new version.

**Comment by TomasVotruba** [comment]
Ref #94

**Comment by TomasVotruba** [comment]
Resolved by #114

**Comment by tomasfejfar** [comment]
Verified. Works fine now ❤️
